private-templates-service/client/src/components/Designer/Designer.tsx

```diff
@@ -133,8 +146,12 @@ function initDesigner(): ACDesigner.CardDesigner {
 }
 
 function onSave(designer: ACDesigner.CardDesigner, props: DesignerProps): void {
-  if (JSON.stringify(props.templateJSON) !== JSON.stringify(designer.getCard()) || props.sampleDataJSON !== designer.sampleData) {
-    props.updateTemplate(props.templateID, props.version, designer.getCard(), designer.sampleData, props.templateName);
+  if(props.templateID === ""){
```

Comment:
Doesn't this mean you can save the default template

Reply:
yes but this is only considered on the first save and the user is re directed to the save modal where they can change the name and tags.

Reply:
yeah but doesnt that mean the modal opens and they try to save but it wont save because its the default template